debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresReplicationConnection.java

```diff
@@ -291,14 +290,18 @@ private PGReplicationStream startPgReplicationStream(final LogSequenceNumber lsn
     public synchronized void close() {
         try {
             super.close();
-        } catch (SQLException e) {
+        } catch (Throwable e) {
```

Comment:
No code after "}" please :) Sorry for the nitpicking, but the quicker we all adjust to the agreed-on style, the less friction down the road. Thanks!